Lib/webbrowser.py

```diff
@@ -542,11 +542,13 @@ def register_standard_browsers():
         # First try to use the default Windows browser
         register("windows-default", WindowsDefault)
 
-        # Detect some common Windows browsers, fallback to IE
-        iexplore = os.path.join(os.environ.get("PROGRAMFILES", "C:\\Program Files"),
-                                "Internet Explorer\\IEXPLORE.EXE")
+        # Detect some common Windows browsers, fallback to Edge
+        edge = os.path.join(os.environ.get("PROGRAMFILES(x86)", "C:\\Program Files (x86)"),
+                            "Microsoft\\Edge\\Application\\msedge.exe")
+        edge32 = os.path.join(os.environ.get("PROGRAMFILES", "C:\\Program Files"),
```

Comment:
The use of `edge32` for the 64-bit version is a bit odd. Any particular reason to not call this `edge64`?

Reply:
Microsoft Edge 64 bit version is still installed in the program files(x86) directory by default. 

The install directory was not updated for  64 bit edge, possibly to keep it consistent with Chrome which also did not update the directory.

However on 32 bit versions of windows, there is no program files(x86) directory or environment variable, so edge is installed in 'program files' directory by default.

I recognise this is counter intuitive but I thought to keep the code consistent with how it works. What are your thoughts? Should I add a comment to clarify?

Reply:
Thanks for this feedback, I've clarified with some comments. Would you mind taking another look?

Reply:
Makes perfect sense now you've explained it!